server/models/remote_provider.go

```diff
@@ -289,25 +289,13 @@ func (l *RemoteProvider) fetchUserDetails(tokenString string) (*User, error) {
 		return nil, ErrDataRead(err, "User Data")
 	}
 
-	up := &UserPref{
-		Preferences: &Preference{
-			AnonymousUsageStats:  true,
-			AnonymousPerfResults: true,
```

Comment:
Any reasons for this, does it prevent saving performance results, because we want to be greedy to have statistics recorded.

Reply:
No reason, I thought preferences could be controlled on the UI side.

Reply:
Yes, they are controlled in UI. This is also fine

Reply:
I added it back since, you wanted to enforce certain behaviour or preferences .